src/components/nav/components/searchBox/SearchBox.js

```diff
@@ -0,0 +1,137 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import { useHandleStatus } from "../../../../hooks/useHandleStatus";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faMagnifyingGlass } from "@fortawesome/free-solid-svg-icons";
+import { faAngleUp } from "@fortawesome/free-solid-svg-icons";
+
+const SearchBox = () => {
+  const { isStatus, handleStatus } = useHandleStatus();
+
+  const [selectStatus, setSelectStatus] = useState("지역");
+
+  const innerSelectorName = event => {
+    const { innerText } = event.target;
+    setSelectStatus(innerText);
+    handleStatus();
+  };
+
+  const innerPlaceHolder = () => {
+    if (selectStatus === "지역") {
+      return "동,읍,면을 입력해 주세요";
+    } else if (selectStatus === "지하철") {
+      return "지하철역 이름을 입력해 주세요";
+    } else if (selectStatus === "대학교") {
+      return "대학교 이름을 입력해 주세요";
+    }
+  };
```

Comment:
객체를 만들어두고 매핑해서 사용하는 게 더 관리하기 좋을 것 같습니다.
```suggestion
  const LOCATION_DESC = {
    "지역" : "동,읍,면을 입력해 주세요",
    "지하철" : "지하철역 이름을 입력해 주세요",
    "대학교" : "대학교 이름을 입력해 주세요",
  };
```

Reply:
넵 반영했습니다. 감사합니다